libdrgn/dwarf_index.c

```diff
@@ -1032,6 +1051,7 @@ void drgn_dwarf_index_read_module(struct drgn_dwarf_index_update_state *state,
 		{
 			struct drgn_dwarf_index_cu cu = {
 				.module = module,
+				.scn = scn,
```

Comment:
I hate to add fields to structs that are used a lot, but I think I will need some way to tell if a CU is a TU or not, and this is the simplest way I could think of doing that. If you have ideas for alternatives, let me know!